swim/member.go

```diff
@@ -50,11 +50,11 @@ type Member struct {
 }
 
 // suspect interface
-func (m Member) address() string {
+func (m *Member) address() string {
```

Comment:
The member has a lock embedded but it is used nowhere I believe. Best to double check that... And maybe we should change that in different PR